### ISSUE TITLE
About the energy file

### ISSUE THREAD

**Issue by kangmintong** [JosephKJ/OWOD#16]
Hi JosephKJ. According to your codes, energy distribution is generated in the execution of validation file. But validation is executed on the dataset which contains unknown instances. I think it's not proper to use information of unknown instances. Do I have any misunderstanding?
Thanks.

**Comment by JosephKJ** [comment]
We are not using any labelled information from the unknown instances. Can you please refer to the line where you got confused?

**Comment by kangmintong** [comment]
My understanding is that the validation set of task 1 should not contain images which come from task2,3,4 even if they're not labelled. I'm not familiar with open world learning so I wander if it's right in this setting.

**Comment by CuberrChen** [comment]
Does this approach still depend on a particular data set because of the need to learn the distribution of energy?

**Comment by JosephKJ** [comment]
ORE is trained on data from VOC and COCO. Please read Sec 5.1 of the paper. The framework is general to handle any dataset.
